src/synthesize.jl

```diff
@@ -202,7 +202,7 @@ You can provide either or both of:
 """
 function format_A_X(metallicity::Real=0.0, abundances::Dict=Dict();
                               solar_relative=true, solar_abundances=asplund_2020_solar_abundances)
-    if (1 in keys(abundances)) || ("H" in keys(abundances))
+    if (((1 in keys(abundances)) && (abundances[1] != 0)) || (("H" in keys(abundances)) && (abundances["H"] != 12)))
```

Comment:
Don't you also want to allow the cases where `abundances["H"]` is `0` or `abundances[1]` is `12` (depending on solar_relative)?  This could be moved down to `clean_abundances` is built up.  Then you wouldn't have to worry about whether the key is `"H"` or `1`.